### PR TITLE
Add test coverage on multipart upload

### PR DESCRIPTION
**Pull request by lucyge2022**
### What changes are proposed in this pull request?

1. add multipart upload tests for correct part ordering
2. add log4j configs to show details such as filename/linenumber/threadid for better tracking

### Why are the changes needed?

improve test coverage on large multipart upload corruption 

### Does this PR introduce any user facing changes?

No

### CODE REVIEW

File: conf/log4j.properties [hunk 1]
```diff
@@ -31,7 +31,7 @@ log4j.appender.=org.apache.log4j.varia.NullAppender
 log4j.appender.Console=org.apache.log4j.ConsoleAppender
 log4j.appender.Console.Target=System.out
 log4j.appender.Console.layout=org.apache.log4j.PatternLayout
-log4j.appender.Console.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
+log4j.appender.Console.layout.ConversionPattern=%d{ISO8601} %-5p %c{1}[%t](%F:%L) - %m%n
```
Comment (jiacheliu3) [inline conf/log4j.properties:34]: move to a separate PR?

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks!

**Review by Jackson-Wang-7 — APPROVED** [review]
LGTM, thanks!

**Comment by yuzhu** [comment]
alluxio-bot, merge this please
